doc/api/fs.md

```diff
@@ -5008,6 +5034,20 @@ changes:
 Changes the ownership on a symbolic link then resolves the `Promise` with
 no arguments upon success.
 
+### `fsPromises.lutimes(path, atime, mtime)`
+<!-- YAML
+added: v10.0.0
+-->
+
+* `path` {string|Buffer|URL}
+* `atime` {number|string|Date}
+* `mtime` {number|string|Date}
+* `gid` {integer}
+* Returns: {Promise}
+
+Changes the file system timestamps on a symbolic link then resolves the `Promise` with
```

Comment:
Same comment as above + a nit: resolving a promise means the promise's "fate" is locked not that it is "finished".

A promise starts off as "pending" and then "settles" by transitioning into either fulfilled or rejected.

A promise resolves by tracking another promise _or_ a value. So for example:

```js
const p1 = fetch('...');
const p2 = Promise.resolve(p1); // p2 is a resolved promise but not a settled one
``` 

Normally this is not a significant difference but since this is literally our docs - we should try to be accurate :]